src/PaymentManager/Payment/PayU.php

```diff
@@ -324,7 +326,10 @@ public function setAuthorizedData(array $authorizedData)
     }
 
     /**
-     * @inheritdoc
+     * @param PriceInterface|null $price
+     * @param string|null $response
+     *
+     * @return Status
      */
     public function executeDebit(PriceInterface $price = null, $response = null)
```

Comment:
normally the second parameter is `reference` and is a `?string`, here it looks like was used as a "shortcut/quick fix" to pass the response array but needs to be refactored